pages/common/git-svn.md

```diff
@@ -10,6 +10,10 @@
 
 `git svn rebase`
 
+- Fetch updates from remote SVN repository without changing the GIT HEAD:
```

Comment:
does GIT have to be in uppercase? I thought the name of the variable was just "HEAD".